src/components/OptionsList/BaseOptionsList.js

```diff
@@ -77,6 +80,63 @@ class BaseOptionsList extends Component {
         Log.hmmm('[OptionsList] scrollToIndex failed', info);
     }
 
+    /**
+     * This function is used to deterministically compute the layout of any given item in our list.
+     * We need to implement it so that we can programmatically scroll to any item in the list.
+     * If we don't, then we can only scroll to items in the virtual render window of the SectionList.
```

Comment:
NAB
```suggestion
     * We need to implement it so that we can programmatically scroll to items outside of the virtual render window of the SectionList.
```